client/src/main/java/org/spine3/validate/Validate.java

```diff
@@ -215,6 +216,28 @@ public static Timestamp checkPositive(Timestamp timestamp, String argumentName)
     }
 
     /**
+     * Ensures that the passed int value is positive:
+     *
+     * @param value the value to check
+     * @param argumentName the name of the checked timestamp to be used in the error message
+     * @throws IllegalArgumentException if requirement is not met
+     */
+    public static void checkPositive(int value, String argumentName) {
+        checkParameter(value > 0, argumentName, ErrorMessage.MUST_BE_A_POSITIVE_VALUE);
```

Comment:
Please call `checkPositive()` with long argument. This way you'd have only one method to test.